sass/browser.scss

```diff
@@ -95,3 +95,8 @@ nav.eb-tabs {
     background-size: 55%;
   }
 }
+
+.bene-media-browser-file-name.views-field.views-field-name {
+  background-color: #ebebeb;
+  padding-top: 10px;
+}
```

Comment:
@5adiyah I don't see the browser.css file being updated here - could you add/commit that?